src/sass/_icons-styles.scss

```diff
@@ -1,4 +1,4 @@
-@import url(https://weloveiconfonts.com/api/?family=brandico|entypo|fontelico|iconicfill|iconicstroke|maki|openwebicons|typicons|zocial);
+@import url(http://weloveiconfonts.com/api/?family=brandico|entypo|fontelico|iconicfill|iconicstroke|maki|openwebicons|typicons|zocial);
```

Comment:
Why not https?

Reply:
without changing url icons are not displayed on the Vuestic Icons Page, related issue:
https://github.com/epicmaxco/vuestic-admin/issues/287

Reply:
meh, https://github.com/TimPietrusky/weloveiconfonts/issues/42
@asvae , what do we do, use http?

Reply:
@jean-moldovan Sure. Tiny hack.